### ISSUE TITLE
format of straight line equation with negative gradient

### ISSUE THREAD

**Issue by lcabrini** [mo-c/moses-cpp-learning#34]
If I use the following:
x1: 0
y1: -9
x2: 3
y2: 3

I get the following:
The gradient of the line is :4
The intercept of the line is : -9
The equation of the line is :y =4x+-9

But 
y = 4x +- 9 
does not look goot. If the intercept is negative it should change the operator instead, as so:
y = 4x - 9

**Comment by mo-c** [comment]
Can you please help me with how to sole this bug?

**Comment by lcabrini** [comment]
Sure. What you need to do is create a variable of type char (meaning it holds a single letter). You can call it operator, or op, or something like that.

If the intercept is negative, then you set the operator variable to a '-', otherwise you set it to '+'. You also need to convert the intercept to its absolute value, |b|. That is done via the std::abs() function which is available in the <cmath> header.

So, to summarize:

    #include <cmath>
    
    char operator = '+';
    if (b < 0) {
        operator = '-'
        b = std:abs(b);
    }

This code will ensure that the value of b is always positive and that the variable operator contains eight a plus sign or a minus sign. From here, try to figure out how to output it in the equation.
